cmd/protoc-gen-netlify-cms/main.go

```diff
@@ -375,8 +375,9 @@ func inferField(
 		}
 		proto.Merge(field, fieldAnnotation)
 	}
-	if field.Widget.WidgetType != nil {
-		return field, true // if a widget is specified, no further inference
+	if field.Widget.WidgetType != nil &&
+		!(protoField.Desc.Kind() == protoreflect.StringKind && !protoField.Desc.IsList()) {
```

Comment:
The original idea was that either you specify a custom widget and then early-exit here, or do the inference below - with this change this assumption doesn't hold anymore.

Reply:
Yes, but only for the name field.

The annoying part is for `name` we change the label from `Name` to `Resource Name`, and required set to `true`,  and these fields has nothing to do with what widget it is.